pallets/system/src/header_builder.rs

```diff
@@ -158,3 +180,86 @@ pub trait HostedHeaderBuilder {
 		)
 	}
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub struct AvailExtrinsic {
```

Comment:
Maybe it makes sense to move this struct to da-primitives and reuse it in both header builder and rpc call?